### PR TITLE
Fixed bug with Shath'Gral faiths not being allowed Ritual Cannibalism.

### PR DESCRIPTION
**Pull request by PaperPlate-Github**
## Note:
Commented out the line that blocked the religion from being able to use it for custom faiths. I did the same for any other lines with the comment "They have it by default" since I assumed that meant religions under that should be able to use the tenet.

I only commented them out since the issue was vague in what it wanted changed.
#114

## Changelog:
- Custom Shath'gral, Deathbringer, and Burning Legion religions may now use the Ritual Cannibalism tenet.

## Developer changelog:
- The religions were marked as not being able to use the tenet in the core_tenets file.
- There could be other tenets that were marked as unavailable to religions that should be able to use them.

### CODE REVIEW

File: common/religion/doctrines/00_core_tenets.txt [hunk 1]
```diff
@@ -1650,7 +1650,7 @@
 		is_shown = {
 
 			# Warcraft
-			evil_public_religion_trigger = no	#They have it by default
+			#evil_public_religion_trigger = no	#They have it by default
```
Comment (zumbak04) [inline common/religion/doctrines/00_core_tenets.txt:1653]: You fixed it wrong. They ALREADY have it from their special doctrine. Thus they shouldn't be able to pick and their religions shouldn't have it.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/religion/doctrines/00_core_tenets.txt (hunk 1 above)
Comment (PaperPlate-Github) [reply common/religion/doctrines/00_core_tenets.txt:1653]: I've re-disabled the tenets.
Cannibalism isn't included in the special doctrine. Should I enable that tenet for evil religions to pick or should I add it to the special doctrine? Or is it ok that they can't pick the tenet when making heresies, and the issue is instead that G'huuni should be given a different tenet?

Comment (zumbak04) [reply common/religion/doctrines/00_core_tenets.txt:1653]: Nevemind, I think I figured it out. So, there's `doctrine_end_justifies_means` I talked about that all evil religions have, but Ritual Cannibalism isn't included there. Thus we should allow picking Ritual Cannibalism and that's all.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**